wooCommerce_scripts/Scripts/WooCommerceUtility.py

```diff
@@ -22,6 +23,22 @@ def assert_status_code(self):
         assert self.status_code == self.expected_status_code, f"if bad status code."\
           f"Expected {self.expected_status_code}, Actual status code: {self.status_code},"\
           f"URL: {self.url}, Response Json: {self.rs_json}"
+    def post(self, wc_endpoint, params=None, expected_status_code=200):
+
```

Comment:
It would be nice to have logging here.
Add two lines.
```
logger.info(f"Calling POST {wc_endpoint}")
logger.debug(f"Payload for POST {wc_cendpoint}: {params}")
```

Reply:
Do similar for the 'get' & 'delete' functions too.